### PR TITLE
Add FFT plot to vizapp

### PR DESCRIPTION
**Pull request by wbenoit26**
Adds a plot of the FFT of the whitened strain to the vizapp. Having some trouble actually getting the vizapp to load, so I need to work that out.

### CODE REVIEW

**Comment by EthanMarx** [comment]
are you pointing to the 100 year O3 dataset or a smaller dataset? It can take a decent chunk of time (5-10 minutes)  to load the full O3 analysis

**Comment by wbenoit26** [comment]
I get the message that the vizapp has loaded, but I just have a blank screen in my browser. Do I also need to wait there?

**Comment by EthanMarx** [comment]
Yeah it can take some time. I think it's b/c all of the background data is being sent to the browser. We don't ever really investigate events below some detection statistic threshold so it might be worth trying setting a lower limit which should get rid of the bulk of the events

File: projects/plots/plots/vizapp/main.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def main(
 
 
 def cli(args=None):
-    parser = jsonargparse.ArgumentParser(parser_mode="omegaconf")
+    parser = jsonargparse.ArgumentParser()
```
Comment (EthanMarx) [inline projects/plots/plots/vizapp/main.py:16]: I think we need this `parser_mode` to be able to specify environment variables in the config. Was something going wrong?

**Review by EthanMarx — COMMENTED** [review]
Overall looks good, excited to see the if the `ffts` provide insight. One question about removing `omega_conf` parser mode

File: projects/plots/plots/vizapp/main.py (hunk 1 above)
Comment (wbenoit26) [reply projects/plots/plots/vizapp/main.py:16]: Yeah, it was telling me that the option was found, or something like that. I meant to just remove it for testing though, I'll put it back

Comment (EthanMarx) [reply projects/plots/plots/vizapp/main.py]: You might need to install `omegaconf` if it isn't

Comment (wbenoit26) [reply projects/plots/plots/vizapp/main.py]: Oh, I didn't realize it was a separate library. Let me add that to the `pyproject.toml`.

**Comment by wbenoit26** [comment]
@EthanMarx Is this good to go now that the parser mode has been fixed?

**Review by EthanMarx — APPROVED** [review]

**Pull request merged**